server/src/Dao/RevisionDao.php

```diff
@@ -61,6 +63,16 @@ public function getUsersCommonPages( array $users ) {
 			->having( 'count(distinct rev_user) > 1' )
 			->setParameter( ':users', $users, Connection::PARAM_STR_ARRAY );
 
+		if ( $startDate ) {
+			$query->andWhere( 'rev_timestamp >= :start_date' )
+				->setParameter( ':start_date', date( 'Ymdhmi', $startDate ) );
+		}
+
+		if ( $endDate ) {
+			$query->andWhere( 'rev_timestamp <= :end_date' )
+				->setParameter( ':end_date', date( 'Ymdhmi', $endDate ) );
```

Comment:
You should check to make sure `date` doesn't return `FALSE`
https://secure.php.net/manual/en/function.date.php

Or (and this is what I typically do), use `DateTime`
https://secure.php.net/manual/en/datetime.construct.php
which will throw an exception on failure
and then `format()`
https://secure.php.net/manual/en/datetime.format.php

If you do it this way, you can also create the object in the method that calls this one, that way you can use a typehint. :)